src/components/Table/GridTable.tsx

```diff
@@ -404,13 +408,25 @@ export function GridTable<R extends Kinded, S = {}, X extends Only<GridTableXss,
   const firstRowMessage =
     (noData && fallbackMessage) || (tooManyClientSideRows && "Hiding some rows, use filter...") || infoMessage;
 
+  const borderless = style === beamFixedStyle;
+  const tableStyle = style === beamFixedStyle ? "fixed" : style === beamFlexibleStyle ? "flexible" : undefined;
```

Comment:
This seems really hard-coded... is there a way we can move this away from "we have two global `const`s that are special"?

Seems like at least we could do `GridStyle.style = "fixed" | "flexible"`...

Although I was hoping that "beam fixed" and "beam flexible" would just be presets that set lower-level `GridStyle` settings...

So, like `GridStyle.borderless: boolean` seems like one?... Maybe also `GridStyle.typeScale`?

Reply:
Yea, I was trying to think of another way of doing this. I did want to eventually have this be completely dictated by GridStyle, but was hoping for feedback on an approach here, so thanks!
When I first looked at GridStyle then I kind of saw it as mostly css/`Properties`, and moved away from that. But I like your suggestion and will update to use this approach. Then the `beamFixedStyles` would be something like:
```
const beamFixedStyle = { borderless: true, typeScale: "xs", ... } 
```

Reply:
Cool! ...for `borderless`, I wonder if we should repeat some of those `PresentationContext` props within `GridStyle` itself, e.g. have `GridStyle.borderless`, or we should have like `GridStyle.presentationContext`? 

My only disclaimer about the 2nd approach is that I think we probably still want to provide some "all tables get it for free" default presentation settings, e.g. borderless, hide labels, etc., although we would probably use like a base presentation settings for all tables, and then just override those with the current table's `style.presentationSettings`? Maybe...

Reply:
I think this is done now per your suggestions. Thanks again for the insightful and helpful feedback!